index.js

```diff
@@ -83,6 +85,10 @@ function PinoColada () {
     return emojiLog[level] + ' '
   }
 
+  function formatNs (name) {
+    return chalk.cyan(name)
```

Comment:
if the above comment makes sense, let's keep it to `chalk.blue`, same as in `formatName()`